flow-plugins/flow-gradle-plugin/src/main/kotlin/com/vaadin/gradle/VaadinPlugin.kt

```diff
@@ -61,4 +69,16 @@ public class VaadinPlugin : Plugin<Project> {
             }
         }
     }
+
+    private fun verifyGradleVersion() {
+        val currentVersion = GradleVersion.current();
+        val supportedVersion =
+            GradleVersion.version(GRADLE_MINIMUM_SUPPORTED_VERSION)
+        if (currentVersion < supportedVersion) {
```

Comment:
Minor comment.

Maybe? as `7.6.1` shall be supported as well if I understand (from the naming)
```suggestion
        if (currentVersion <= supportedVersion) {
```

Reply:
Ah, you are right, it was a mistake from me,  I miss see this logic :) that is why it is good that we have covering tests :D. 
We can close this, the condition works well like this! 👍 
Thanks!